content/en/docs/reference/command-line-tools-reference/kube-apiserver.md

```diff
@@ -610,7 +610,7 @@ kube-apiserver [flags]
 <td colspan="2">--feature-gates mapStringBool</td>
 </tr>
 <tr>
```

Comment:
Don't change this in the PR. This file is auto generated.

Reply:
Fixed